src/app/redux/UserSaga.js

```diff
@@ -245,6 +245,18 @@ function* usernamePasswordLogin2({
             symbol: LIQUID_TOKEN_UPPERCASE,
         }
     );
+
+    // added by realmankwon (2019-06-12) fetch all SCOT stake
+    const all_token_balances = yield call(
```

Comment:
Can you make this call once, and place the SCT one in the right place?

Reply:
sorry, I can understand what you mean.

Reply:
Ah I didn't see this. Because the call above does it for SCT specifically, and then the all_token_balances does the same call but for all.